### PR TITLE
hrefPointers wording improvements

### PR DESCRIPTION
**Pull request by dlax**
Instead of pointing at typos or suggesting clarifications on #386 , I thought that I could just do the changes myself and submit a PR again your working branch. So if you agree and merge this PR, #386 will then get the new commits.

### CODE REVIEW

File: jsonschema-hyperschema.xml [hunk 1]
```diff
@@ -679,8 +679,8 @@
                 </t>
                 <figure>
                     <preamble>
-                        Recall that a Relative JSON Pointer which is not also a regular
```
Comment (handrews) [inline jsonschema-hyperschema.xml]: Ugh, neither of these is correct.  regular JSON Pointers are also valid Relative JSON Pointers.  I had it backwards.  Yours is correct but not what I meant :-P  I'll fix it and update the main PR (possibly by merging this as-is and then amending it, need to read the paragraph re-write below and think on it).

Comment (handrews) [reply jsonschema-hyperschema.xml]: Although really there's some debate over this.  Some people seem to think that it's necessar to say "JSON Pointer or Relative JSON Pointer" which seems horribly verbose to me.  Not sure what to do with it.  I was just going to change the Relative JSON Pointer spec to say what I want it to say and re-submit it as draft-andrews-relative-json-pointer-00, probably :-P

Comment (dlax) [reply jsonschema-hyperschema.xml]: Yes this wasn't clear for me. The approach you took in #386 is better, thanks.

**Comment by dlax** [comment]
Closing as (interesting) changes got incorporated in the upstream PR.

**Pull request closed without merging**